vpdq/vpdq-realese.py

```diff
@@ -0,0 +1,63 @@
+from distutils.command.sdist import sdist
+from platform import release
+import shutil
+from pathlib import Path
+import os
+import subprocess
+import argparse
+
+
+DIR = Path(__file__).parent
+PARENTDIR = Path(__file__).parents[1]
+
+SETUP = "setup.py"
+MANIFEST = "MANIFEST.in"
+DIST = "dist"
+
+
+def get_argparse() -> argparse.ArgumentParser:
+    ap = argparse.ArgumentParser(description=__doc__)
+    ap.add_argument(
+        "-r",
+        "--release",
+        help="If release, will return the source distribution in the root dir.",
+        action="store_false",
```

Comment:
blocking: This is super surprising based on the name of the command.

Since it's storing a value of "false", I would expect this to be a negation - "no-release"

Did you mean store_true here?